bin/cli.js

```diff
@@ -79,7 +79,9 @@ For more information, see https://webpack.js.org/api/cli/.`);
 			type: "boolean",
 			alias: "colors",
 			default: function supportsColor() {
-				return require("supports-color").supportsColor;
+				if (process.stdout.isTTY === true) {
```

Comment:
The PR says to not use supports-color. So why here it's the opposite?

Reply:
` don't use supports-color on non-tty`

Reply:
Yeah I see my mistake :D